share/spack/gitlab/cloud_pipelines/stacks/e4s/spack.yaml

```diff
@@ -259,6 +259,13 @@ spack:
       - gptune
 
     mappings:
+      - match:
+          - kokkos +rocm amdgpu_target=gfx90a
+        runner-attributes:
+          tags: [ "uo-gilgamesh-docker-rocm", "gfx90a" ]
+          variables:
+            CI_JOB_SIZE: large
+
```

Comment:
What will happen on protected pipelines then?  I think it will mean this `kokkos` won't get built and signed, correct?  Similar to why we are reworking how the power stack behaves.

Reply:
I hadn't thought about that. I see your point. I don't know. What can be done about it?

Reply:
@kotfic was going to reach out to you about having UO runners join in as signers of protected binaries.  Until that happens, the only way I can think of is to keep any specs like this one in their own stack that doesn't get signed (but treated like the mac and power specs).

Reply:
I'll reach out to @kotfic and find out what is involved in getting it set up so that UO runners can be part of protected pool. Depending on how long that will take, we can either wait until that is complete or, if you don't object, I could submit a separate PR to create a standalone, mini-ROCm stack that initially only contains `kokkos +rocm` -- the purpose of which will be to test out these new runners with AMD GPU's mapped in. Let me check on this and I will let you know what I find out.

Reply:
Sounds great, I don't object at all.  

It would be nice to use the slightly refactored `.gitlab-ci.yml` from the reworked e4s power stack, but nothing touching the `gitlab-ci.yml` can be merged until the broken specs on develop get fixed.